.azure-pipelines/sync-aliases.yml

```
@@ -0,0 +1,42 @@
+# Variable 'AzureToken' and 'BotAccessToken' was defined in the Variables tab
+resources:
+  repositories:
+  - repository: self
+    type: git
+    ref: main
+
+trigger: none
+
+jobs:
+- job: UpdateJson
+  displayName: Update Json
+  steps:
+  - checkout: self
+  - pwsh: |
+      ./tools/Github/ParseWiki2Json.ps1 -AzureToken $(AzureToken)
+    displayName: Update fabricbot.json file locally
+      
+  - task: PowerShell@2
+    displayName: Git commit and push
+    inputs:
+      targetType: inline
+      script: |
+        git pull origin main
+        git config --global user.email "65331932+azure-powershell-bot@users.noreply.github.com"
+        git config --global user.name "azure-powershell-bot"
+        git checkout -b "Change_fabricbot.json_aliases"
+        
+        git add .
+        git commit -m "Use pipeline to sync fabricbot22.json"
+        
+        git remote set-url origin https://$(BotAccessToken)@github.com/Azure/azure-powershell.git;
+        git push origin Change_fabricbot.json_aliases --force
+  
+  - pwsh: |
+      $Title = "Change Fabricbot.json According To Wiki Page"
```

```suggestion
      $Title = "Sync Fabricbot.json According To ADO Wiki Page"
```
Fabricbot.json is 'F' uppercase or lowercase?